controllers/controllers.topics.js

```diff
@@ -0,0 +1,8 @@
+const {selectTopics} = require('../models/models.topics')
+
+const getTopics = (req, res, next) => {
+    selectTopics().then((topics) =>
+        res.status(200).send({ topics }))
+}
+
```

Comment:
I would include a catch block here, if there is an error this needs somewhere to go so it would also be appropriate to add a 500 error handler at this point.